app/controllers/v1/rules_controller.rb

```diff
@@ -25,12 +25,16 @@ def resource
       Rule
     end
 
+    def serializer
+      RuleSerializer
+    end
+
     def search_by_id
-      ::Pundit.policy_scope(User.current, ::Rule).friendly.find(params[:id])
+      pundit_scope.friendly.find(params[:id])
     end
 
     def search_by_ref_id
-      rule = Rule.latest.where(
+      rule = pundit_scope.latest.where(
```

Comment:
This change caused a test to fail - I'm quite sure we always want to filter records through the pundit policy. In this case, canonical rules are intended to be returned (canonical rules are those assigned to any canonical profile). The rule pundit policy allows this.